### ISSUE TITLE
Add plugin ID as command parameter or as config within the Resources/store folder

### ISSUE THREAD

**Issue by momocode-de** [FriendsOfShopware/FroshPluginUploader#21]
If I wan't to use the plugin uploader for multiple plugins I have to edit the PLUGIN_ID in the .env file every time I swtich the plugin. I think the .env file is not the correct place for it and it would make more sense if I can add the plugin ID as command parameter. 

Or better to a config file within the Resources/store folder. So I can add the plugin ID once and then it's versioned and read out automatically.

**Comment by shyim** [comment]
Hey,
the reason for the env is, that we use that in the Travis CI. So its updated on any git change.

Maybe we could add a global argument to override it?

**Comment by momocode-de** [comment]
I have decided to fork the project and adjust the code for my needs. I made a "plugin:init" command with plugin path and plugin ID as parameter. This command will download the resources to the store folder of the plugin. And then I have added the plugin ID to the store.json. So all other commands only have the plugin path as parameter and they read the plugin ID from the store.json and not from the .env.
